File: Pertemuan 6/belajarnestedif.py
umur = input("printMasukkan umur Anda: ")
umur = int(umur)
if (umur >=0 and umur <=5):
    print("Anda masih balita ")
    print("Masukkan umur orang tua Anda: ")
    umur_orangtua = input("Masukkan umur orang tua Anda: ")
    if (int(umur_orangtua)< 15):
        print("Sebaiknya orang tua perlu pengasuh anak")
    else:
        print("Orang tua bisa mengasuh anak sendiri")
elif (umur > 5 and umur <= 19):
    print("Anda sudah remaja")
else:
    print("Umur Anda tidak valid")

File: Pertemuan 6/BelajarNestedIf.java
import java.util.Scanner;

public class BelajarNestedIf {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Masukkan umur Anda: ");
        int umur = scanner.nextInt();
        if (umur <= 5 && umur >= 0) {
            System.out.print("Anda masih balita ");
            System.out.print("Masukkan umur orang tua Anda: ");
            int umurOrangTua = scanner.nextInt();
            if (umurOrangTua < 15) {
                System.out.print("Sebaiknya Anda meminta pengasuh anak");
            } else {
                System.out.print("Anda bisa mengasuh anak sendiri");
            }
        } else if (umur > 5 && umur <= 19) {
            System.out.print("Anda sudah remaja");
        } else if (umur > 19) {
            System.out.print("Anda sudah dewasa");
        } else {
            System.out.print("Umur Anda tidak valid");
        }
    }
}